mapping/sustainability.yaml

```
@@ -584,8 +584,52 @@
         Disclose who is the climate finance investment intended to benefit \[free text to explain the beneficiaries\] and number of beneficiaries. \[Select from the list:
         Direct
         Indirect\]
-    mapping: ''
-    example: ''
+    mapping: |-
+        Project level: Add a new `benefit` object to the `benefits` array and set `.title` to "Climate finance beneficiaries".
+
+        For direct beneficiaries:
+
+        - Add a `Beneficiary` object to the `.beneficiaries` array.
+          - Set the `.description` to "direct beneficiaries".
+          - Set the number of people to `.numberOfPeople`.
+          - Add the location of the beneficiary population to `.location`
+
+        For indirect beneficiaries:
+
+        - Add a `Beneficiary` object to the `.beneficiaries` array.
+          - Set the `.description` to "indirect beneficiaries".
+          - Set the number of people to `.numberOfPeople`.
+          - Add the location of the beneficiary population to `.location`
+    example: |-
+        {
+          "benefits": [
+            {
+              "title": "Cliamte finance beneficiaries",
+              "beneficiaries": [
+                {
+                  "description": "Direct beneficiaries",
+                  "numberOfPeople": 1000,
+                  "location": {
+                    "id": "1",
+                    "address": {
+                      "countryName": "HT"
+                    }
+                  }
+                },
+                {
+                  "description": "Indirect beneficiaries",
+                  "numberOfPeople": 2000,
+                  "location": {
+                    "id": "1",
+                    "address": {
+                      "countryName": "HT"
+                    }
+                  }
+                }
+              ]
+            }
+          ]
+        }
```

The climate finance investment funds the project so its beneficiaries are the same as the project's beneficiaries.

This change will cause a build error until https://github.com/open-contracting/infrastructure/pull/428 is merged and `0.9-dev` is merged into this branch.

```suggestion
    mapping: See [number of beneficiaries (social module)](social-number-of-beneficiaries)
    example: ''
```

---

suggested change applied, leaving this as unresolved just as a reminder that the build is failing because #428 hasn't been approved and merged yet

---

I've merged `0.9-dev` (including #428) so the build is passing now.